pkg/receive/handler.go

```diff
@@ -258,7 +262,7 @@ type replica struct {
 	replicated bool
 }
 
-func (h *Handler) handleRequest(ctx context.Context, rep uint64, tenant string, wreq *prompb.WriteRequest) error {
+func (h *Handler) handleRequest(ctx context.Context, rep, seriesLimit uint64, tenant string, wreq *prompb.WriteRequest) error {
```

Comment:
The `seriesLimit` field seems to be added to the protobuf, so can we add it once to the request to avoid propagating it to every function?